### ISSUE TITLE
Several jscs-jsdoc rules confilcts with jscs rule

### ISSUE THREAD

**Issue by sameoldmadness** [jscs-dev/jscs-jsdoc#89]
Given `.jscsrc`

```json
{
    "plugins": [
        "jscs-jsdoc"
    ],
    "jsDoc": {
        "checkRedundantParams": true,
        "checkRedundantReturns": true,
        "requireParamTypes": true,
        "requireReturnTypes": true
    },
    "disallowNewlineBeforeBlockStatements": true
}
```

and javascript file

```js
(function () {
    BEM.DOM.decl('yandex-metrika', {
        onSetMod: {
            js: {
                inited: function () {
                    BEM.channel('metrika').on('goal', this._reachGoal.bind(this));
                }
            }
        },

        /**
         * @protected
         * @param {Object} event
         * @param {string} goal
         */
        _reachGoal: function (event, goal) {
            
        }
    });
}());
```

jscs gives me an error:

```bash
, and *
         * @protected
         * @param {Object} event
         * @param {string} goal
          should have at most 2 line(s) between them at static/desktop.blocks/yandex-metrika/yandex-metrika.js :
     7 |                }
     8 |            }
     9 |        },
------------------^
    10 |
    11 |        /**


1 code style error found.
```

When I disable `jscs` rule or _every_ `jscs-jsdoc` rule, linting successfully passes.

**Comment by zxqfox** [comment]
Hm. Are you sure that the problem in these rules? Looks like there can be another rule that modifing JSFile data.

```
$ cat qwe.test.js
(function () {
    BEM.DOM.decl('yandex-metrika', {
        onSetMod: {
            js: {
                inited: function () {
                    BEM.channel('metrika').on('goal', this._reachGoal.bind(this));
                }
            }
        },

        /**
         * @protected
         * @param {Object} event
         * @param {string} goal
         */
        _reachGoal: function (event, goal) {

        }
    });
}());
$ cat .jscsrc
{
    "plugins": [
        "jscs-jsdoc"
    ],
    "jsDoc": {
        "checkRedundantParams": true,
        "checkRedundantReturns": true,
        "requireParamTypes": true,
        "requireReturnTypes": true
    },
    "disallowNewlineBeforeBlockStatements": true
}
$ npm list | grep jscs
qqq@0.0.0 /home/alex/repos/jscs-jsdoc/qqq
├─┬ jscs@1.12.0
│ ├── esprima-harmony-jscs@1.1.0-templates
└─┬ jscs-jsdoc@0.4.5
$ jscs qwe.test.js
No code style errors found.
$ jscs qwe.test.js -x
No code style errors found.
```

**Comment by sameoldmadness** [comment]
```bash
$ cat .jscsrc 
{
    "plugins": ["jscs-jsdoc"],
    "jsDoc": { "requireParamTypes": true },
    "disallowMultipleLineBreaks": true
}
$ cat static/desktop.blocks/yandex-metrika/yandex-metrika.js
BEM.DOM.decl('yandex-metrika', {
    onSetMod: {
    },

    /**
     * @protected
     * @param {Object} event
     * @param {string} goal
     */
    _reachGoal: function (event, goal) {
    }
});
$ grep version node_modules/jscs/package.json 
  "version": "1.12.0",
$ grep version node_modules/jscs-jsdoc/package.json 
  "version": "0.4.5",
$ node_modules/.bin/jscs static/desktop.blocks/yandex-metrika/yandex-metrika.js
, and *
     * @protected
     * @param {Object} event
     * @param {string} goal
      should have at most 2 line(s) between them at static/desktop.blocks/yandex-metrika/yandex-metrika.js :
     1 |BEM.DOM.decl('yandex-metrika', {
     2 |    onSetMod: {
     3 |    },
--------------^
     4 |
     5 |    /**


1 code style error found.
```

**Comment by sameoldmadness** [comment]
just in case:

```bash
$ cat /etc/*release*
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.2 LTS"
NAME="Ubuntu"
VERSION="14.04.2 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.2 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
$ node -v
v0.11.13
$ npm -v
2.1.10
```

**Comment by sameoldmadness** [comment]
oh shi... I messed with jscs rule : )

the correct one is `disallowMultipleLineBreaks`.

**Comment by zxqfox** [comment]
```
$ node -v
v0.11.13
```
Wow.

Yeah, thanks! I've got that error. Looks like it's an issue in jscs. Investigating...

**Comment by zxqfox** [comment]
Thanks again. Landed in v0.4.6.

**Comment by sameoldmadness** [comment]
Great.
